libpod/container_internal.go

```diff
@@ -264,7 +264,7 @@ func (c *Container) handleRestartPolicy(ctx context.Context) (_ bool, retErr err
 	if c.ensureState(define.ContainerStateRunning, define.ContainerStatePaused) {
 		return false, nil
 	} else if c.state.State == define.ContainerStateUnknown {
-		return false, errors.Wrapf(define.ErrInternal, "invalid container state encountered in restart attempt!")
+		return false, errors.Wrapf(define.ErrInternal, "invalid container state encountered in restart attempt")
```

Comment:
```suggestion
		return false, fmt.Errorf("invalid container state encountered in restart attempt: %w", define.ErrInternal)
```

Reply:
Yes lets do this as a global change separate from this PR.